modules/builtin/src/content-types/_utils.js

```diff
@@ -2,7 +2,7 @@ const URL = require('url').URL
 const path = require('path')
 
 function isBpUrl(str) {
-  const re = /^\/api\/.*\/bots\/.*\/media\/.*/
+  const re = /^api\/.*\/bots\/.*\/media\/.*/
```

Comment:
Shouldn't this be optional? I mean, won't this break current URL starting with /api ?